photocollage/gtkgui.py

```diff
@@ -25,12 +25,12 @@
 
 import cairo
 import gi
-gi.require_version('Gtk', '3.0')  # noqa: E402
-from gi.repository import Gtk, Gdk, GObject, GdkPixbuf  # noqa: I202
-from six.moves import urllib  # Python 2 backward compatibility
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk, Gdk, GObject, GdkPixbuf  # noqa: E402, I202
+from six.moves import urllib  # Python 2 backward compatibility  # noqa: E402
 
-from photocollage import APP_NAME, artwork, collage, render
-from photocollage.render import PIL_SUPPORTED_EXTS as EXTS
+from photocollage import APP_NAME, artwork, collage, render  # noqa: E402
+from photocollage.render import PIL_SUPPORTED_EXTS as EXTS  # noqa: E402
```

Comment:
I guess you're adding those `E402` because of the recent `flake8` change.

Instead, can you move `gi.require_version(...)` + `from gi.repository import Gtk, ...` after the other imports? This way, only the last line would need the `noqa: E402` (which makes sense, because it's the one that needs an import tweak).

Reply:
> I guess you're adding those `E402` because of the recent `flake8` change.

Yes.

> Instead, can you move `gi.require_version(...)` + `from gi.repository import Gtk, ...` after the other imports? This way, only the last line would need the `noqa: E402` (which makes sense, because it's the one that needs an import tweak).

Good idea, updated! Also changed the `I202` to `I100`.